packages/wrangler/src/dev.tsx

```diff
@@ -600,24 +603,22 @@ export async function startDev(args: StartDevOptions) {
 			);
 		}
 
-		let experimentalAssetsOptions = processExperimentalAssetsArg(args, config);
-		if (experimentalAssetsOptions) {
-			args.forceLocal = true;
-		}
-
-		/*
-		 * - `config.legacy_assets` conflates `legacy_assets` and `assets`
-		 * - `args.legacyAssets` conflates `legacy-assets` and `assets`
-		 */
 		if (
 			(args.legacyAssets || config.legacy_assets) &&
-			experimentalAssetsOptions
```

Comment:
keeping experimental assets and legacy assets validation separate here, for consistency with the rest of our code base